modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-source-to-image-build.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-source-to-image-build-in-a-network-restricted-environment_{context}']
+= Creating source-to-image build in a network restricted environment
+
+You can create a `source-to-image` build in a network restricted environment by mirroring the images required by `source-to-image` BuildStrategy. 
```

Comment:
```suggestion
You can create a `source-to-image` build in a network restricted environment by mirroring the images required by the `source-to-image` BuildStrategy. 
```

Reply:
I think you need a "the" here, and also same questions as above about BuildStrategy.